packages/paste-design-tokens/tokens/themes/dark/global/text-color.yml

```diff
@@ -155,31 +155,15 @@ props:
   color-text-decorative-10:
     value: "{!palette-gray-20}"
     comment: Text color with no semantic meaning, used for decorative purposes only. Should generally be used with matching decorative background and/or border tokens.
-    text_contrast_pairing:
```

Comment:
No need to declare the pairings at the theme level. The pairings are set at the system level, so every theme should meet the same token pair requirements as they are used in the same places across the theme.